src/components/input-field/partial-styles/_readonly.scss

```diff
@@ -31,29 +31,32 @@
             }
         }
 
-        .mdc-text-field__icon.trailing-icon {
-            display: flex;
-            justify-content: flex-end;
-            align-items: center;
-
-            pointer-events: auto;
-            cursor: pointer;
+        .mdc-text-field__icon {
```

Comment:
> prevent leading icons becoming transparent in `readonly`

Hmm… this seems like something that ought to be fixed in chip-set rather than here? 🤔

Reply:
I think it's fine here. Because we don't want the leading or trailing icons to become transparent and look disabled in `readonly` mode. It's also true in input field.

Reply:
Yeah, but if we fix it here, it doesn't apply to chip-set in readonly? If we fix it in chip-set, it should automatically apply in all components using chip-set.